hal/src/main/native/athena/REVPH.cpp

```diff
@@ -224,6 +224,8 @@ HAL_REVPHHandle HAL_InitializeREVPH(int32_t module,
   hph->previousAllocation = allocationLocation ? allocationLocation : "";
   hph->hcan = hcan;
   hph->controlPeriod = kDefaultControlPeriod;
+  hph->desiredSolenoidsState = 0;
+  hph->versionInfo = 0;
```

Comment:
This won't zero out structs. `std::memset(&hph->versionInfo, 0, sizeof(hph->versionInfo);`

Reply:
*sigh* It's been quite a day, thank you for putting up with this.

Reply:
Wouldn't `hph->versionInfo = {}` work too?

Reply:
Yeah that would too.